test/server/cards/02_SHD/units/MandalorianWarrior.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Mandalorian Warrior', function () {
+    integration(function () {
+        describe('Mandalorian Warrior\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mandalorian-warrior'],
+                        groundArena: ['protector-of-the-throne', 'battlefield-marine'],
```

Comment:
We typically try to add units to make sure all of the filters are working correctly. In this case, could you please add:
- A deployed Mandalorian leader to confirm that leader units are targetable
- An enemy Mandalorian unit to confirm that enemy units are targetable